src/config/rcxml.c

```diff
@@ -826,6 +826,34 @@ entry(xmlNode *node, char *nodename, char *content)
 		} else {
 			wlr_log(WLR_ERROR, "Invalid value for <resize popupShow />");
 		}
+	} else if (!strcasecmp(nodename, "top.area.tablet")) {
+		double top = atof(content);
```

Comment:
There is a lot of duplication in the four branches here. Could we write a helper function like `rc.tablet.box.x = get_non_neg_double(content)`. Could add error message as second argument. Return 0 if negative.